frontend/src/components/regions/DueTimeFormModal/Presentation.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+
+import TextField from '@material-ui/core/TextField';
+import Button from '@material-ui/core/Button';
+import Grid from '@material-ui/core/Grid';
+
+const useStyles = makeStyles({
+
+    dueTimeButton: {
+        marginTop: "0.3rem",
+        marginRight: "0.3rem",
+    },
+});
+
+const DueTimeForm = (props) => {
+
+  const { dueTime } = props;
+  const defaultValue = dueTime ? dueTime.split('').slice(0, 16).join('') : "";
+  const classes = useStyles();
+
+  return (
+      <form novalidate>
+        <TextField
+          id="datetime-local"
```

Comment:
This is in the develop branch
Copy pasting between branches is a no-go. We use git to get info from one branch to another.